flexmeasures/ui/utils/view_utils.py

```diff
@@ -110,6 +110,15 @@ def clear_session():
         del session[skey]
 
 
+def set_session_variables(*var_names: str):
+    """Store request values as session variables, for a consistent UX across UI page loads.
+
+    >>> set_session_variables("event_starts_after", "event_ends_before", "chart_type")
+    """
+    for var_name in var_names:
+        session[var_name] = request.values.get(var_name)
```

Comment:
I think that if var_name isn't in the request, we should not set the session value.